### DISCUSSION TITLE
Using Godot 4

### DISCUSSION THREAD

**Discussion by paulloz** [NEW-CYLANDIA/little-warioware#11] [General]
With Godot 4 getting closer and closer to having an RC soon, I'm wondering if it's worth it to switch to 4 directly. Mainly because GDScript V2 is way, way better.

**Comment by iznaut** [comment]
oooh yeah, i'm all for that!

would be really nice just to be on the latest and greatest from the start rather than having to migrate down the road

**Comment by paulloz** [comment]
Cool, made a quick version on the `godot-4` branch. Haven't brought every feature from `master` yet.

**Comment by iznaut** [comment]
@paulloz fyi - i'm gonna start putting more time into this project in the lead up to GDC....would really like to have something to promote at the Little Mario talk!

lmk if there's anything i can do to help with the Godot 4 upgrade 🙂

**Comment by paulloz** [comment]
I'm second guessing myself. Because of limitations about web exports in Godot 4, we might want to stay on 3.5 for now 🤔

**Comment by TaylorAnderson** [comment]
Oh I didn't notice this was here! Yeah, it does seem like the web export stuff is a little experimental at this point for Godot 4, it might be best to wait until that's all sorted out or 4.1 comes out with a single-thread export that can bypass all this.

**Comment by iznaut** [comment]
sounds good! let's just stick to 3.5 for now and we'll cross the upgrade bridge when we come to it

**Comment by paulloz** [comment]
Yep, I concur.

**Comment by iznaut** [comment]
it still seems like Godot 4 web exports are a little junk so we're putting this off for the foreseeable future
